lib/property_sets/active_record_extension.rb

```diff
@@ -184,6 +184,38 @@ def update_property_set_attributes(attributes)
           end
         end
       end
+
+      def update_columns(attributes)
+        attrs = if delegated_property_sets?
+          attributes.reject{|k,_| self.class.delegated_property_set_attributes.include?(k.to_s) }
+        else
+          attributes
+        end
```

Comment:
```suggestion
        if delegated_property_sets?
          attributes = attributes.reject{|k,_| self.class.delegated_property_set_attributes.include?(k.to_s) }
        end
```
... then use `attributes` below